knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -2,6 +2,14 @@ package knativekafka
 
 import (
 	"context"
+	"fmt"
+	mfc "github.com/manifestival/controller-runtime-client"
+	mf "github.com/manifestival/manifestival"
+	"github.com/openshift-knative/serverless-operator/knative-operator/pkg/common"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	"os"
```

Comment:
Please order. `goimports` is your friend.

Reply:
done